tests/utils/compute_groundtruth.cpp

```diff
@@ -381,9 +381,8 @@ template <typename T>
 int aux_main(const std::string &base_file, const std::string &query_file, const std::string &gt_file, size_t k,
              const diskann::Metric &metric, const std::string &tags_file = std::string(""))
 {
-    size_t npoints, nqueries, dim, npoints_filt;
+    size_t npoints, nqueries, dim;
 
```

Comment:
why has base_data gone? was not used is it?

Reply:
yes. they are not used. so I could see the warnings. So I just remove them to fix warnings